src/EditorFeatures/Core.Wpf/SignatureHelp/Controller.Session_ComputeModel.cs

```diff
@@ -101,8 +101,8 @@ internal partial class Session
                             currentModel.Provider == provider &&
                             currentModel.GetCurrentSpanInSubjectBuffer(disconnectedBufferGraph.SubjectBufferSnapshot).Span.Start == items.ApplicableSpan.Start &&
                             currentModel.Items.IndexOf(currentModel.SelectedItem) == items.SelectedItemIndex &&
-                            currentModel.ArgumentIndex == items.ArgumentIndex &&
-                            currentModel.ArgumentCount == items.ArgumentCount &&
```

Comment:
renamed these to make the distinction clear.  'argument count' refferred to how many explicit argument syntax nodes are present in the code.  'argument index' was very badly named.  It indicated 'this is the *parameter* in the actual symcol we think the user is currently on'

Reply:
note: the pr is almost entirely this rename.  i call out anything that isn't just that.